.golangci.yml

```diff
@@ -0,0 +1,67 @@
+---
+# Almost all linters; some of them are optional.
+
+linters-settings:
+  depguard:
+    rules:
+      main:
+        deny:
+          - pkg: errors
```

Comment:
do you think we need this rule here?

Reply:
This project used both, `errors` and `pkg/errors`. So with this rule we will have at least consistency. But approach should be revisited, maybe it makes sense to use only standard `errors`